main/views.py

```diff
@@ -57,8 +57,58 @@ def get_queryset(self):
         return self.model.objects.filter(not_after__gt=now, not_before__lt=now)
 
 
-class HomeView(LoginRequiredMixin, _SystemMessageView):
+class SeasonalCoverImageMixin:
```

Comment:
Okay so this addition is 100% unnecessary. I just wanted to have a bit of fun during a boring meeting. 

It adds a seasonal-appropiate cover-image automatically. 

The winter and autumn covers are the ones we already had, the spring and summer ones are new. 

Note 1: I made the old 'summer' version the 'autumn' one. Mostly because it had people in winter-coats in it, so it wasn't really summer. 
Note 2: I found out that our attributions we had before were wrong! We listed the person who added the image to the image-bank, not the actual photographer themselves. The ones listed here now are correct.